udata_gouvfr/templates/licences.html

```diff
@@ -0,0 +1,272 @@
+{% extends theme("base.html") %}
+
+{% set meta = {
+    'title': 'Licences',
+    'description': 'Licences de réutilisation des informations publiques.',
+    'keywords': [
+        'Licence Ouverte',
+        _('Licence'),
+        'Service Public de la Donnée',
+    ],
+} %}
+
+{% set body_class="licences" %}
+
+{% block content %}
+{% cache cache_duration, 'licenses', g.lang_code %}
+<section class="content">
+    <div class="container">
+        <div class="row">
+            <h1 class="col-xs-12">Licenses de réutilisation</h1>
+        </div>
+    </div>
+</section>
+
+<section class="content">
+    <div class="container">
+        <div class="row">
+            <article class="col-xs-12 col-sm-8 col-md-9">
+
+
+<p>
+    Afin d’éviter la prolifération des licences, la
+    <a href="https://www.legifrance.gouv.fr/affichTexteArticle.do?cidTexte=JORFTEXT000033202746&amp;idArticle=JORFARTI000033203004&amp;categorieLien=cid" >
+    loi pour une République numérique
+    </a>
+    a prévu la création d’une liste, fixée par décret, de licences
+    qui peuvent être utilisées par les administrations pour la
+    réutilisation à titre gratuit de leurs informations publiques,
+    qu’il s’agisse de données ou de code source d’un logiciel
+    (<a href="https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000034504991&amp;cidTexte=LEGITEXT000031366350&amp;dateTexte=29991231">article
+    D.323-2-1 du code des relations entre le public et l’administration
+    (CRPA)</a>).
+</p>
+<p>
+    La présente page référence toutes les licences utilisables
+    à ce titre par les administrations. Elle pourra être tenue
+    à jour au gré des évolutions de versions de ces textes, et
+    des éventuelles évolutions du décret.
+</p>
+<p>
+    La première partie de cette page dresse la liste des licences
+    applicables aux informations publiques ainsi qu’aux codes sources
+    utilisables par l’administration
+</p>
+<p>
+    La deuxième partie de cette page dresse la liste des licences
+    spéciales homologuées avec le périmètre applicable de
+    l’homologation (information publique spécifique, administration
+    spécifique, ou autre) au titre de l’<a href="https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000034504993&amp;cidTexte=LEGITEXT000031366350&amp;dateTexte=29991231">article D. 323-2-2</a> du CRPA.
+</p>
+
+<h2>
+    Les licences pouvant être utilisées par les administrations
+</h2>
+<ul>
+    <li>
+    Les licences applicables aux « informations publiques » (données, documents…) 
```

Comment:
There is consistently one level of indent per level of imbricated unordered lists.